File: Examples/test-suite/php/casts_runme.php
<?php

require "tests.php";

// No new functions
check::functions(array());
// New classes
check::classes(array('A','B'));
// No new vars
check::globals(array());

# Make sure $b inherits hello() from class A
$b=new B();
$b->hello();

check::done();

File: Examples/test-suite/lua/casts_runme.lua
require("casts")

local a = casts.A()
assert(a ~= nil)
assert(swig_type(a) == "A *")
a:hello()


-- Make sure b inherits hello() from class A
local b = casts.B()
assert(b ~= nil)
assert(swig_type(b) == "B *")
b:hello()
